custom/src/main/java/co/elastic/otel/dynamicconfig/CentralConfig.java

```diff
@@ -314,6 +316,22 @@ void update(String configurationValue, OpampManager opampManager)
     }
   }
 
+  public static final class InferSpans extends ConfigOption {
+    InferSpans() {
+      super("infer_spans", "true");
+    }
+
+    @Override
+    void update(String configurationValue, OpampManager opampManager)
+        throws IllegalArgumentException {
+      // TODO when setProfilerInterval() returns the old value, use that for 'true'
```

Comment:
Can we deal with this directly in this PR or are we blocked on https://github.com/open-telemetry/opentelemetry-java-contrib/pull/2354 to be included in a release ?